backend/apps/notifications/channels/email_channel.py

```
@@ -0,0 +1,66 @@
+        message = getattr(notification, "message", "")
+
+        template_html = None
+        template_txt = None
+        if notif_type:
+            # <notification_type>_email.html
+            try:
+                template_html = render_to_string(
+                    f"notifications/{notif_type}_email.html",
+                    {"title": title, "message": message, "recipient": recipient, "meta": getattr(notification, "meta", {})},
+                )
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)